src/ProgressOnderwijsUtils/Data/CascadedDelete.cs

```diff
@@ -20,13 +20,14 @@ public static class CascadedDelete
             ParameterizedSql initialTableAsEntered,
             bool outputAllDeletedRows,
             [CanBeNull] Action<string> logger,
+            [CanBeNull] Func<string, bool> stopCascading,

```

Comment:
Deze api is niet voldoende om bijv. cycles te doorbreken, en een beetje non-obvious voor "respect on cascade null instead of manual cascade delete".

Reply:
Waar zit jij aan te denken dan?

Reply:
Naja, als je een api wilt die cycles kan doorbreken moet ie meer parameters mee krijgen zodat ie zelf de rijen mag afhandelen uit de to-be-deleted temptable (en dus waarschijnlijk zelf een delegate moet enqueuen op de deletion stack), en als je alleen wilt dat "respect on cascade null" geldt dan is het misschien makkelijker om dat te doen.

Maar om er ergens mee te kunnen beginnen zou ook deze API kunnen, maar dan wel met een naam die duidelijker is; "stopCascading" zegt niet zoveel - wat is de string input, wat precies "stopt", etc.  Bijv. een parameter `string[] tablesToNeverDeleteFrom`